### DISCUSSION TITLE
How to join in team as a contributor

### DISCUSSION THREAD

**Discussion by jni2000** [open-quantum-safe/liboqs#1738] [General]
Sorry to post this question here if not appropriate. I am new and would like to join in as a contributor. With over 20 years software development experiences using C/C++/Java/Python and PKI background, I hope I can contribute to these great projects.

**Comment by baentsch** [comment]
@jni2000 glad to hear that: You'd surely be very welcome as contributor. Please see [the OQS governance  doc](https://github.com/open-quantum-safe/liboqs/blob/main/GOVERNANCE.md#contributors) highlighting the "rules of engagement". Depending on your background, you may want to check issues labelled "Help wanted" or "Good first issue" to see where more hands on the keyboard are particularly wanted. In turn, if you have new issues you yourself deem sensible to work on, please create those in the suitable sub project for wider discussion.

**Reply by jni2000** [reply]
thanks, @baentsch, will do.

**Reply by baentsch** [reply]
@jni2000 would you be OK closing this discussion? Or do you have any further questions/suggestions so you can begin to contribute?
